src/EditorFeatures/CSharpTest/BraceMatching/CSharpBraceMatcherTests.cs

```diff
@@ -747,5 +749,41 @@ static void Main(string[] args)
 
             await TestAsync(code, expected);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.BraceMatching), Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)]
+        public async Task StartTupleDeclaration()
+        {
+            var code = @"public class C { $$(int, int, int, int, int, int, int, int) x; }";
+            var expected = @"public class C { (int, int, int, int, int, int, int, int[|)|] x; }";
+
+            await TestAsync(code, expected, TestOptions.Regular.WithTuplesFeature());
```

Comment:
If you are testing this by `ParseOptions`, does `TestOptions.Regular.WithTuplesFeature()` test both regular C# and C# interactive cases? Just noting that if the ordinary flow tests both the regular and script options, then this feature specific flow should do the same.